packages/template-retail-react-app/app/components/store-locator-modal/stores-list.jsx

```diff
@@ -19,72 +19,77 @@ import {
 const StoresList = ({storesInfo}) => {
     const intl = useIntl()
 
-    return storesInfo.map((store, index) => {
-        return (
-            <AccordionItem key={index}>
-                <Box margin="10px">
-                    {store.name ? <Box fontSize="lg">{store.name}</Box> : ''}
-                    <Box fontSize="md" color="gray.600">
-                        {store.address1}
-                    </Box>
-                    <Box fontSize="md" color="gray.600">
-                        {store.city}, {store.stateCode ? store.stateCode : ''} {store.postalCode}
-                    </Box>
-                    {store.distance !== undefined ? (
-                        <>
-                            <br />
-                            <Box fontSize="md" color="gray.600">
-                                {store.distance} {store.distanceUnit}{' '}
-                                {intl.formatMessage({
-                                    id: 'store_locator.description.away',
-                                    defaultMessage: 'away'
-                                })}
-                            </Box>
-                        </>
-                    ) : (
-                        ''
-                    )}
-                    {store.phone !== undefined ? (
-                        <>
-                            <br />
-                            <Box fontSize="md" color="gray.600">
-                                {intl.formatMessage({
-                                    id: 'store_locator.description.phone',
-                                    defaultMessage: 'Phone:'
-                                })}{' '}
-                                {store.phone}
-                            </Box>
-                        </>
-                    ) : (
-                        ''
-                    )}
-                    {store?.storeHours ? (
-                        <>
-                            {' '}
-                            <AccordionButton color="blue.700" style={{marginTop: '10px'}}>
-                                <Box fontSize="lg">
+    return storesInfo !== undefined ? (
+        storesInfo.map((store, index) => {
```

Comment:
Minor: In React 18 (which we use now), it's ok to return `undefined` to render nothing. So we can simplify the code like this:
```jsx
return storesInfo?.map(() => {...})
```